extensions/homeassistant/src/components/states.tsx

```diff
@@ -227,6 +228,12 @@ function getIcon(state: State): Image.ImageLike | undefined {
     return { source: source, tintColor: PrimaryIconColor };
   } else if (e.startsWith("weather")) {
     return { source: weatherConditionToIcon(state.state) };
+  } else if (e.startsWith("fan")) {
+    let source = "fan.png";
+    if (state.attributes.percentage == 0) {
```

Comment:
Maybe it would be also good to detect the state also via the entity state and make the tintColor gray when the state is unavailable.